test/models.json

```diff
@@ -1,4 +1,16 @@
 [
+  {
+    "type":     "hailo",
+    "target":   "resnet_v1_18.hn",
+    "source":   "https://hailo-tappas.s3.eu-west-2.amazonaws.com/netron/resnet_v1_18.hn",
+    "format":   "HailoNN (hn)"
```

Comment:
Would it make sense to rename to `Hailo NN`? The naming pattern is "Name v1.0",  no brackets.